Bottom: d436bbfe44414dff6aba237f362dc6e17d0a175e
Top:    f1c2cbfdae62e28c86090f488e6f52417b0b05e8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:32 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
